modules/selenium/testcontainers/selenium/__init__.py

```diff
@@ -48,9 +48,16 @@ class BrowserWebDriverContainer(DockerContainer):
     """
 
     def __init__(
-        self, capabilities: str, image: Optional[str] = None, port: int = 4444, vnc_port: int = 5900, **kwargs
+        self,
+        capabilities: dict[str, Any],
+        options: Optional[ArgOptions] = None,
```

Comment:
Isn't the docstring generation failing because we are missing the import for `ArgOptions` here?